Sources/PortalSwift/Core/Keychain/PortalKeychain.swift

```diff
@@ -229,12 +231,15 @@ public class PortalKeychain {
       let shares = try decoder.decode(PortalKeychainClientShares.self, from: data)
 
       return shares
-    } catch KeychainError.itemNotFound(_) {
+    } catch {
       self.logger.debug("PortalKeychain.getShares() - Attempting to read from legacy share data...")
       // Handle backward compatibility with legacy Keychain data
       do {
         // Before multi-wallet support was added
         let signingShareValue = try keychain.getItem("\(clientId).share")
+        
+        print("🚨 Signing Share: \(signingShareValue)")
```

Comment:
I don't think you should be printing this. remove this log.

Reply:
✅ Removed